### PR TITLE
Import updates

### PR DESCRIPTION
**Pull request by dczulada**

### CODE REVIEW

File: importer/importer.go [hunk 1]
```diff
@@ -641,7 +648,7 @@ func set_patient_expired(patient *models.Record, xmlNode xml.Node) {
 // create status code. then set status code from status if necessary
 func set_status_code(entry *models.Entry, status string) {
 	entry.StatusCode = map[string][]string{}
-	if status != "" { // only set a status code if status is not empty
+	//if status != "" { // only set a status code if status is not empty
```
Comment (ghost) [inline importer/importer.go]: Can we delete these lines instead of just commenting them out?

File: models/coded.go [hunk 2]
```diff
@@ -40,9 +40,9 @@ func (c *Coded) AddCodeIfPresent(codeElement xml.Node) {
 
 	//extract nullFlavor from attribute if it is NA
 	nullFlavorAttribute := codeElement.Attribute("nullFlavor")
-	nullFlavorValueSet := codeElement.Attribute("valueSet")
-	if nullFlavorAttribute != nil && nullFlavorValueSet != nil {
-		code = nullFlavorValueSet.String()
+	nullFlavorValueSet := codeElement.Attributes()
```
Comment (ghost) [inline models/coded.go:43]: How come we are pulling out all the attributes and referencing the `valueSet` attribute later? Why not just extract the one we need on the codeElement.Attribute() call?

File: models/entry.go [hunk 3]
```diff
@@ -32,6 +32,7 @@ type Reference struct {
 	Type           string `json:"type,omitempty"`
 	ReferencedType string `json:"referenced_type,omitempty"`
 	ReferencedID   string `json:"referenced_id,omitempty"`
+	ExportedRef	   string `json:"exported_ref,omitempty"`
```
Comment (ghost) [inline models/entry.go]: Need to run gofmt on this file so the whitespace is aligned.

**Review by ghost — CHANGES_REQUESTED** [review]

File: models/coded.go (hunk 2 above)
Comment (dczulada) [reply models/coded.go:43]: I couldn't get codeElement.Attribute("valueSet") to work with namespace prefixes.

File: models/entry.go (hunk 3 above)
Comment (tstrass) [reply models/entry.go]: done

File: models/coded.go (hunk 2 above)
Comment (ghost) [reply models/coded.go:43]: oh I see. you might be able to add the namespace but i'm ok with that work around

**Review by tstrass — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**